src/EnergyPlus/ScheduleManager.cc

```diff
@@ -3030,15 +3060,35 @@ namespace ScheduleManager {
 				EMin = MMField;
 			}
 
+			if ( interpolationKind == ScheduleInterpolation::Linear ) {
+				totalMinutes = ( EHr - SHr ) * 60 + ( EMin - SMin ) + 1;
+				if ( totalMinutes == 0 ) totalMinutes = 1; // protect future division
+				if ( Count == 1 ) {
+					StartValue = Numbers( Count ); //assume first period is flat
+					EndValue = Numbers( Count );
+				} else {
+					StartValue = EndValue;
+					EndValue = Numbers( Count );
+				}
+				incrementPerMinute = ( EndValue - StartValue ) / totalMinutes;
+				curValue = StartValue + incrementPerMinute;
```

Comment:
Is this within a minute by minute loop?  Shouldn't curValue be equal to the StartValue plus the incrementPerMinute times the number of minutes?  I'll keep looking to figure out what's going on, but leaving this as a placeholder for now.

Reply:
This is just before the beginning of the minute by minute loop. It is just setting the initial value of the curValue which should be the value for minute 1.